runtime/onert/backend/gpu_cl/CMakeLists.txt

```diff
@@ -5,8 +5,6 @@ if(NOT ARMCompute_FOUND)
   return()
 endif(NOT ARMCompute_FOUND)
 
-# find_package(abseil REQUIRED)
-
```

Comment:
if it's not making error, or duplicates, how about enabling it. or put corresponding one.
gpu_cl requires abseil is true, defining relation seems reasonable